ReleaseNotes/eng/Altibase_Migration_Center_7_14_Release_Notes.md

```diff
@@ -0,0 +1,264 @@
+| ------------------------------------------------------------ | ------------------------ |
+| Altibase 4.3.9 or higher<br />CUBRID 8.4.1 ~ 9.3.5 (ISO-8859-1, UTF-8 charset) <br/>Informix 11.50 <br />Microsoft SQL Server 2005 ~ 2012<br />Oracle Database 9i ~ 11g <br />Oracle MySQL 5.0 ~ 5.7 <br />Oracle TimesTen 7.0, 11.2 <br />Tibero 4 SP1 ~ 6<br/>PostgreSQL 9.5.3 | Altibase 6.5.1 or higher |
+
+<br/>
+
+# 2. Release Information
+
+This section summarizes new features, fixed bugs, and changes in Migration Center 7.14.
+
+## 2.1 New Features
+
+### BUG-50652	Support User to Modify Select Condition Entered in Reconcile Step Before Executing in Run Step
+
+When extracting data from the source database, a function that allows only data that meets a specific condition to be selectively extracted and migrated has been added. The Select condition can be modified in "Select Editing" during the Reconcile step, or edited directly by the user in the TableCondition.properties file after completing the Reconcile step.
+
+<br/>
+
+## 2.2 Bug-Fixes
+
+### BUG-50263	Data Type Mapping Update: BINARY_DOUBLE of Oracle, TimesTen, and Tibero is Now Mapped to Altibase DOUBLE, Replacing the Previous VARCHAR Mapping
+
+The BINARY_DOUBLE types of Oracle, TimesTen, and Tibero are compatible with the Altibase DOUBLE type, so the default data mapping type need to be changed. However, Oracle, TimesTen, and Tibero support special values of NaN(Not a Number) and INF(Infinity), while Altibase does not. For these values, data loss can occur.
+
+### BUG-50821	Altibase to Oracle Migration Feature is No Longer Supported
+
+Altibase to Oracle migration feature is no longer provided in accordance with company policy.
+
+### BUG-50827 	Data Type Mapping Update: The TimesTen Binary Type is Now Mapped to Altibase BYTE, Replacing the Previous BLOB Mapping
+
+The default data mapping type of TimesTen Binary type is more suitable for BYTE type than Altibase BLOB type, which is a large binary data type, considering data size range and compatibility. Therefore, the data mapping type of TimesTen Binary type is changed from Altibase BLOB type to BYTE type.
+
+### BUG-51034	Provide Empty String Data Conversion Function for Migration to Altibase
+
+Since Altibase does not distinguish between Empty String and Null data, unexpected results may occur when migrating records with Empty String. To prevent this, a function of converting Empty String data into a form suitable for Altibase is added to the Migration Option.
+
+### BUG-51035	Provide Conversion Function for Not Null & Default ''(Empty String) Columns During Migration to Altibase
+
+Since Altibase does not distinguish between Empty String and Null data, columns with Not Null and Default ''(Empty String) settings in other DBMS cannot be migrated as they are. To solve this problem, the Migration Option provides the function of converting Not Null & Default ''(Empty String) column into DDL suitable for Altibase.
```

Comment:
```suggestion
Since Altibase does not distinguish between Empty String and Null data, columns with Not Null and Default ''(Empty String) settings in other DBMS cannot be migrated as they are. To solve this problem, the Migration Option provides the function of converting Not Null & Default ''(Empty String) column into DDL statements suitable for Altibase.
```
빈 문자열 조정 기능이 엄밀히 말하면 DDL 구문을 조정하는 것이기 때문에 statement를 추가했습니다. 가급적 DDL은 CREATE, ALTER 등의 명령어로, DDL statement는 DDL을 이용한 sql 구문 그 자체로 나누어서 작성하고 있습니다. 해당 버그의 성격을 고려했을때 DDL 구문을 알티베이스에 맞게 변환하는 것이기 때문에 제안 드립니다.

Reply:
> 빈 문자열 조정 기능이 엄밀히 말하면 DDL 구문을 조정하는 것이기 때문에 statement를 추가했습니다. 가급적 DDL은 CREATE, ALTER 등의 명령어로, DDL statement는 DDL을 이용한 sql 구문 그 자체로 나누어서 작성하고 있습니다. 해당 버그의 성격을 고려했을때 DDL 구문을 알티베이스에 맞게 변환하는 것이기 때문에 제안 드립니다.

넵 내용 확인하였습니다. 수정하였습니다.